files/default/rsync_ca_cwlogs.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+
+set -e
+
+# get the list of ca hosts
+ca_list=`curl -s -u $CA_STATUS_USER_PASS $CA_STATUS_URL | jq -r '.[].address'`
+ca_count=`echo $ca_list | wc -w`
+echo "Fetched $ca_count entries"
+
+# rsync the logs
+for ca_host in $ca_list; do
+    echo "fetching from $ca_host"
+    rsync -e "ssh -i $CA_KEY_FILE" -azvh --quiet --inplace \
+        --exclude "messages.*" --exclude "mhpearl.log.*" \
```

Comment:
How do you rotate the logs? Pearl logs are not rotated at midnight.(I can change it)

Reply:
no rotation needed on the local side. the rsync is only fetching `messages` and `mhpearl.log`. when a rotation happens on the device side the rsync will inplace update the local files with the new remote ones. there's no appending happening so the local logs don't grow and thus don't need to be rotated.